src/index.js

```diff
@@ -10,16 +13,12 @@ var Menu = require('electron').Menu;
  *     misspelled, `false` if it is spelled correctly or is not text.
  *   @property {Array<String>=[]} spellingSuggestions - An array of suggestions
  *     to show to correct the misspelling. Ignored if `isMisspelled` is `false`.
- *
+ * @param {Array} mainTemplate - optional
+ * @param {Object} suggestionsTemplate - optional
```

Comment:
Can you add a little description of these parameters like "The template of always-present menu items.", "The template of spelling suggestion items"?

Also it looks like `suggestionsTemplate` is an `Array` too?